stash/todo/src/main/java/org/axonframework/messaging/eventhandling/LegacyEventHandlingComponent.java

```diff
@@ -22,6 +22,10 @@
 import org.axonframework.messaging.core.MessageStream;
 import org.axonframework.messaging.core.QualifiedName;
 import org.axonframework.messaging.core.unitofwork.ProcessingContext;
+import org.axonframework.messaging.eventhandling.replay.ResetContext;
```

Comment:
Curious: Why add code to this todo class?
If anything, I would've expected any remaining replay/reset specific code to be removed from the todo module (not anything added, actually).